app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -821,7 +821,7 @@ class AdvancedFilter extends React.Component {
    */
   renderSearchStatement = (filter, index, value, additionalFilterOption) => {
     // compute tooltip for specific search cases
-    let tooltip = '';
+    let tooltip = filter.tooltip !== undefined ? filter.tooltip : '';
```

Comment:
you should just be able to do
```suggestion
    let tooltip = filter.tooltip || '';
```

Reply:
Done. Thank you!